boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -103,6 +103,53 @@ config REGULATOR
 
 endif # !TRUSTED_EXECUTION_SECURE
 
+if BOARD_SERIAL_BACKEND_CDC_ACM
+
+config USB_DEVICE_PRODUCT
+	default "Thingy:53 Application"
+
+config USB_DEVICE_VID
+	default 0x1915
+
+config USB_DEVICE_PID
+	default 0x530C
+
+config USB_DEVICE_STACK
+	default y
+
+config USB_CDC_ACM
+	default y
+
+config UART_CONSOLE
+	default CONSOLE
+
+config USB_DEVICE_INITIALIZE_AT_BOOT
+	default y if !MCUBOOT
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
+	default SHELL
+
+config USB_DEVICE_REMOTE_WAKEUP
```

Comment:
That seems not to work, because `CONFIG_USB_NRFX` implies this symbol and it eventually ends up being enabled in applications configurations (e.g. machine learning). It could lead to issues on some of the USB hosts (host could suspend USB hoping that device could wake up the host if needed, but the wakeup would never occur because application does not trigger it)

Reply:
You could either disable it in `_defconfig` or remove the imply from driver's code (I would suggest removing the imply, because it requires application interaction to wake up the USB host so application should be aware that the feature is enabled)

Reply:
Created upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/54488
+ cherry-picked here as fromlist for now